### PR TITLE
add statosio

### PR DESCRIPTION
**Pull request by a6b8**
- fix tables
- add script tag with sri
- json new line

### CODE REVIEW

**Comment by MattIPv4** [comment]
FYI, in future you don't need to close the PR and open a new one, the old one was updated and was fine to leave open for a second review 👍

File: statosio/change-style-to-dark-mode/index.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
+# Change style to dark-mode
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-customize-400.jpg)<br>
+[Change style to dark-mode](https://d3.statosio.com/tutorials/change-style.html)
```
Comment (MattIPv4) [inline statosio/change-style-to-dark-mode/index.md:6]:
```suggestion
```
I don't think we need this link, as this is just the same tutorial as what we're currently showing?

File: statosio/create-a-simple-point-chart/index.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
+# Create a simple point chart
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-point-400.jpg)<br>
+[Create a simple point chart](https://d3.statosio.com/tutorials/simple-point-chart.html)
```
Comment (MattIPv4) [inline statosio/create-a-simple-point-chart/index.md:6]:
```suggestion
```
No need to link to identical content, just confuses the reader

File: statosio/create-a-simple-stacked-bar-chart/index.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
+# Create a simple stacked bar chart
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-stacked-bar-400.jpg)<br>
+[Create simple stacked bar chart](https://d3.statosio.com/tutorials/simple-stacked-bar-chart.html)
```
Comment (MattIPv4) [inline statosio/create-a-simple-stacked-bar-chart/index.md:6]:
```suggestion
```
No need for link

File: statosio/create-a-simple-stacked-bar-chart/index.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+</body>
+```
+
+- Option explained
+
+| **Name** | **Description** | **Details** | 
+|:---|:---|:---|
+| [showAverage](https://d3.statosio.com/options/show__average.html) | Calculate and show average line | false |
+
+List of all "show" Options: [here](https://d3.statosio.com/options/index.html#show)s
```
Comment (MattIPv4) [inline statosio/create-a-simple-stacked-bar-chart/index.md:183]:
```suggestion
List of all "show" Options: [here](https://d3.statosio.com/options/index.html#show)
```
Random `s`?

File: statosio/select-and-change-position-of-columns/index.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
+# Select and change position of columns
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-select-400.jpg)<br>
+[Select and change position of columns](https://d3.statosio.com/tutorials/select-data.html)
```
Comment (MattIPv4) [inline statosio/select-and-change-position-of-columns/index.md:6]:
```suggestion
```
Remove duplicate content link

File: statosio/select-and-change-position-of-columns/index.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+
+- Option explained
+
+| **Name** | **Description** | **Details** | 
+|:---|:---|:---|
+| [showAverage](https://d3.statosio.com/options/show__average.html) | Calculate and show average line | false |
+
+List of all "show" Options: [here](https://d3.statosio.com/options/index.html#show)
+
+
```
Comment (MattIPv4) [inline statosio/select-and-change-position-of-columns/index.md:226]:
```suggestion
```
Remove extra blank lines at end of file

File: statosio/sort-dataset-by-values/index.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
+# Sort dataset by values
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-sort-400.jpg)<br>
+[Sort dataset by values](https://d3.statosio.com/tutorials/sort-data.html)
```
Comment (MattIPv4) [inline statosio/sort-dataset-by-values/index.md:6]:
```suggestion
```
Rm dupe content link

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
In a few places you use bullet points when they aren't really part of a list -- this seems like an odd style choice for the content? If they're meant to be subheadings, maybe bold would be better?

E.g. `- Option explained`

**Comment by a6b8** [comment]
> In a few places you use bullet points when they aren't really part of a list -- this seems like an odd style choice for the content? If they're meant to be subheadings, maybe bold would be better?
> 
> E.g. `- Option explained`


thanks for the fast reply

- removed bullet points.
- removed links
- removed "s"

File: statosio/create-a-simple-bar-chart/index.md [hunk 8]
```diff
@@ -0,0 +1,182 @@
+# Create a simple bar chart
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-bar-400.jpg)<br>
```
Comment (MattIPv4) [inline statosio/create-a-simple-bar-chart/index.md:5]:
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-bar-400.jpg)
```
br not needed

File: statosio/create-a-simple-bar-chart/index.md [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+|:---|:---|:---|
+| d3.js | [https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js](https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js) | d3.js Library | 
+| statosio.js | [https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js](https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js) | statosio.js Library | 
+| performance.json | [https://d3.statosio.com/data/performance.json](https://d3.statosio.com/data/performance.json) | Dataset |
+
+statosio.js Structure
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```
Comment (MattIPv4) [inline statosio/create-a-simple-bar-chart/index.md:47]: Extra fences need removing

File: statosio/create-a-simple-point-chart/index.md [hunk 10]
```diff
@@ -0,0 +1,187 @@
+# Create a simple point chart
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-point-400.jpg)<br>
```
Comment (MattIPv4) [inline statosio/create-a-simple-point-chart/index.md:5]:
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-point-400.jpg)
```
br not needed

File: statosio/create-a-simple-point-chart/index.md [hunk 11]
```diff
@@ -0,0 +1,187 @@
...
+|:---|:---|:---|
+| d3.js | [https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js](https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js) | d3.js Library | 
+| statosio.js | [https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js](https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js) | statosio.js Library | 
+| performance.json | [https://d3.statosio.com/data/performance.json](https://d3.statosio.com/data/performance.json) | Dataset |
+
+statosio.js Structure
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```
Comment (MattIPv4) [inline statosio/create-a-simple-point-chart/index.md:49]: Extra fences need removing

File: statosio/create-a-simple-stacked-bar-chart/index.md (hunk 3 above)
Comment (MattIPv4) [reply statosio/create-a-simple-stacked-bar-chart/index.md:6]: This link still needs removing

File: statosio/create-a-simple-stacked-bar-chart/index.md [hunk 12]
```diff
@@ -0,0 +1,183 @@
+# Create a simple stacked bar chart
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-stacked-bar-400.jpg)<br>
```
Comment (MattIPv4) [inline statosio/create-a-simple-stacked-bar-chart/index.md:5]:
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-stacked-bar-400.jpg)
```
br not needed

File: statosio/create-a-simple-stacked-bar-chart/index.md [hunk 13]
```diff
@@ -0,0 +1,183 @@
...
+|:---|:---|:---|
+| d3.js | [https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js](https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js) | d3.js Library | 
+| statosio.js | [https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js](https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js) | statosio.js Library | 
+| performance.json | [https://d3.statosio.com/data/performance.json](https://d3.statosio.com/data/performance.json) | Dataset |
+
+statosio.js Structure
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```
Comment (MattIPv4) [inline statosio/create-a-simple-stacked-bar-chart/index.md:48]: extra fences need removing

File: statosio/select-and-change-position-of-columns/index.md [hunk 14]
```diff
@@ -0,0 +1,223 @@
+# Select and change position of columns
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-select-400.jpg)<br>
```
Comment (MattIPv4) [inline statosio/select-and-change-position-of-columns/index.md:5]:
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-select-400.jpg)
```
br not needed

File: statosio/select-and-change-position-of-columns/index.md [hunk 15]
```diff
@@ -0,0 +1,223 @@
...
+|:---|:---|:---|
+| d3.js | [https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js](https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js) | d3.js Library | 
+| statosio.js | [https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js](https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js) | statosio.js Library | 
+| performance.json | [https://d3.statosio.com/data/performance.json](https://d3.statosio.com/data/performance.json) | Dataset |
+
+statosio.js Structure
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```
Comment (MattIPv4) [inline statosio/select-and-change-position-of-columns/index.md:48]: Extra fences need to be removed

File: statosio/sort-dataset-by-values/index.md [hunk 16]
```diff
@@ -0,0 +1,224 @@
+# Sort dataset by values
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-sort-400.jpg)<br>
```
Comment (MattIPv4) [inline statosio/sort-dataset-by-values/index.md:5]:
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-sort-400.jpg)
```
br isnt needed

File: statosio/sort-dataset-by-values/index.md [hunk 17]
```diff
@@ -0,0 +1,224 @@
...
+|:---|:---|:---|
+| d3.js | [https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js](https://cdnjs.cloudflare.com/ajax/libs/d3/6.2.0/d3.js) | d3.js Library | 
+| statosio.js | [https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js](https://cdnjs.cloudflare.com/ajax/libs/statosio/0.9/statosio.js) | statosio.js Library | 
+| performance.json | [https://d3.statosio.com/data/performance.json](https://d3.statosio.com/data/performance.json) | Dataset |
+
+statosio.js Structure
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```
Comment (MattIPv4) [inline statosio/sort-dataset-by-values/index.md:49]: Extra fences need removing

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Tyvm :)

**Review by MattIPv4 — APPROVED** [review]
Sorry for the delay -- all lgtm, tyvm!

**Pull request merged**